Traceback (most recent call last):
  File "/home/asimos/Documents/uff/aula/sd/trabs/rpc/question3/client.py", line 11, in <module>
    print(conn.get_question)
AttributeError: 'Connection' object has no attribute 'get_question'
